docs/cli.md

```diff
@@ -0,0 +1,135 @@
+# ODH Edge CLI
+
+The ODH Edge CLI is a command line interface that allows you to interact with the ODH Edge platform. It is a tool that allows you to manage your ODH Edge models and images. 
+
+## Overview
+
+The ODH Edge CLI uses the Model Registry as the source of truth for all models and images. The Model Registry is a central repository that stores all the metadata about the models and images that are available on the ODH Edge platform. The ODH Edge CLI allows you to interact with the Model Registry to perform various operations such as listing and adding models.
+
+Since building a model image is done via OpenShift Pipelines, all the pipeline parameters that are required to build the model image can be stored in the Model Registry. The ODH Edge CLI allows you to interact with the Model Registry to store and retrieve these pipeline parameters.
+
+> [!NOTE]
+> The Model Registry stores the metadata about the models and images, but it does not store the actual model artifacts.
+> The model artifacts are stored in a separate storage location that is accessible to the ODH Edge platform such as an
+> S3 bucket or a Git repository.
+
+
+## Building ODH Edge CLI
+
+To build the ODH Edge CLI, run the following commands from the root of the repository: 
+
+```bash
+cd cli
+make cli-build
+```
+
+This will create a binary called `odh` in your current directory. You can move this binary to a location in your PATH to make it easier to run the ODH Edge CLI from anywhere.
+
+## Environment Setup
+
+### Prerequisites
+
+- Kubectl access to the OpenShift core cluster.
+- Go (version 1.20 or 1.21, version 1.22 is not supported).
+- [Model registry operator](https://github.com/opendatahub-io/model-registry-operator/releases/tag/v0.1.2) version 0.1.2 locally.
+
+### Installing Model Registry
+
+To install the Model Registry, follow the following steps:
+
+1. Change to the `model-registry-operator-0.1.2` directory:
+
+    ```bash
+    cd model-registry-0.1
+    ```
+2. Install the CRDs into the cluster:
```

Comment:
Should we consider just pointing to the model registry operator repo for the instructions to install that, rather than having our own?